libs/auth/src/inserted_smart_card_auth.ts

```diff
@@ -409,7 +418,11 @@ export class InsertedSmartCardAuth implements InsertedSmartCardAuthApi {
             return {
               ...currentAuthStatus,
               error: undefined,
-              wrongPinEnteredAt: new Date(),
+              lockedOutUntil: computeCardLockoutEndTime(
+                machineState,
+                action.checkPinResponse.numIncorrectPinAttempts
+              )?.getTime(),
+              wrongPinEnteredAt: new Date().getTime(),
```

Comment:
maybe this will become apparent as I get to later commits, but noting that I'm not sure why we need `wrongPinEnteredAt` at all

Reply:
it looks like you're essentially just using this as a boolean flag in the frontend to tell whether the PIN was _just_ incorrectly entered or if it's a lockout attempt from possibly another machine. that works

Reply:
That's right - `wrongPinEnteredAt` actually predates this PR and is more fundamentally used to indicate whether an incorrect PIN has been entered to the frontend.

Code from before this PR: https://github.com/votingworks/vxsuite/blob/fa6a1566ca2d50c4eebe08f70fa46344565042e2/libs/ui/src/unlock_machine_screen.tsx#L81-L83

In addition to the reason you lay out, we also couldn't just rely on `lockedOutUntil` since it's `undefined` for the first few incorrect PIN attempts. We need something else to capture whether an incorrect PIN was entered.